laravel/app/Http/Controllers/UserController.php

```diff
@@ -13,4 +17,47 @@ public function user_logout() {
     public function __construct(){
         $this->middleware('auth');
     }
+
+    public function show($user)
```

Comment:
$userだとUserインスタンスの様に見えてしまうので、idにしましょう
```suggestion
    public function show($id)
```